pkgs/by-name/go/gobang/package.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  stdenv,
+  darwin,
+}:
+
+rustPlatform.buildRustPackage rec {
```

Comment:
Consider removing the `rec` for the `let in` syntax see https://nix.dev/guides/best-practices.html